tests/test.cc

```diff
@@ -1,2 +1,30 @@
+#include "node.h"
+
 #define CATCH_CONFIG_MAIN
 #include "catch.hpp"
+
+TEST_CASE( "Node is checked", "[Node]" ) {
+	Node test (123, {11.2, 22.3, 33.4});
+
+	REQUIRE( test.id() == 123 );
+	REQUIRE( test.coordinates()(2) == 33.4 );
```

Comment:
You need these checks with `Approx` and Tolerance. Like this:
`REQUIRE( test.coordinates()(2) == Approx(33.4).epsilon(Tolerance));`

Reply:
Floating points cannot be checked directly.